src/Uno.UI/Controls/BindableUISwitch.iOS.cs

```diff
@@ -54,17 +55,19 @@ public BindableUISwitch(IntPtr handle)
 			InitializeBinder();
 		}
 
-		public Brush Foreground
+		#region ThumbTintColorBrush DependencyProperty
+
+		//The color used to tint the appearance of the thumb.
+		public Brush ThumbTintColorBrush
 		{
-			get { return (Brush)this.GetValue(ForegroundProperty); }
-			set { this.SetValue(ForegroundProperty, value); }
+			get { return (Brush)this.GetValue(ThumbTintColorBrushProperty); }
+			set { this.SetValue(ThumbTintColorBrushProperty, value); }
 		}
 
-		// Using a DependencyProperty as the backing store for Background.  This enables animation, styling, binding, etc...
-		public static readonly DependencyProperty ForegroundProperty =
-			DependencyProperty.Register("Foreground", typeof(Brush), typeof(BindableUISwitch), new PropertyMetadata(null, (s, e) => ((BindableUISwitch)s).OnForegroundChanged((Brush)e.NewValue)));
+		public static readonly DependencyProperty ThumbTintColorBrushProperty =
```

Comment:
This is a binary breaking change, keep the `ForegroundProperty` and make it an alias to the other property, same for Background.

Reply:
Done Thanks !